lib/chat_api_web/controllers/email_conversation_controller.ex

```diff
@@ -0,0 +1,92 @@
+defmodule ChatApiWeb.EmailConversationController do
+  use ChatApiWeb, :controller
+
+  alias ChatApi.{Conversations, Messages, Customers}
+  alias Customers.{Customer}
+
+  action_fallback(ChatApiWeb.FallbackController)
+
+  @spec create(Conn.t(), map()) :: {:error, Ecto.Changeset.t()} | Plug.Conn.t()
+  def create(
+        conn,
+        %{
+          "account_id" => account_id,
+          "body" => body,
+          "customer_id" => customer_id,
+          "email_address" => email_address
+        } = params
+      ) do
+    subject = Map.get(params, "subject")
+
+    with customer <- Customers.get_customer!(customer_id),
```

Comment:
What's the best practice here for handling a `customer_id` id that doesn't exist? The database throws an exception if this happens, but I didn't see very much error handling in the codebase for this.